File: Assets/VirtualCity/ProtoDefine/RspUpdateHousePartMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]



public class RspUpdateHousePartMessage {
[ProtoMember(1)]
    public int code;
[ProtoMember(2)]
    public string tip;

    public RspUpdateHousePartMessage(int code, string tip) {
        this.code = code;
        this.tip = tip;
    }

    public RspUpdateHousePartMessage() {
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public string getTip() {
        return tip;
    }

    public void setTip(string tip) {
        this.tip = tip;
    }
}
}

File: Assets/VirtualCity/ProtoDefine/Java/building/RspUpdateHousePartMessage.java
package com.kingston.jforgame.server.game.building.message;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.building.BuildingDataPool;
import com.kingston.jforgame.server.game.building.entity.House;
import com.kingston.jforgame.server.game.building.entity.HouseParts;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

import java.util.List;

@MessageMeta(module = Modules.BULIDING,cmd = BuildingDataPool.RSP_UPDATEHOUSEPART)
public class RspUpdateHousePartMessage extends Message {
    @Protobuf(order = 1)
    private int code;
    @Protobuf(order = 2)
    private String tip;

    public RspUpdateHousePartMessage(int code, String tip) {
        this.code = code;
        this.tip = tip;
    }

    public RspUpdateHousePartMessage() {
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getTip() {
        return tip;
    }

    public void setTip(String tip) {
        this.tip = tip;
    }
}
